examples/virtual_sites/vsite_showcase.ipynb

```diff
@@ -536,16 +522,47 @@
     "    water_reference.atoms[0].name = \"O\"\n",
     "    water_reference.atoms[1].name = \"H1\"\n",
     "    water_reference.atoms[2].name = \"H2\"\n",
+    "\n",
     "    # Add ideal TIP5P geometry\n",
+    "    bond_length = Quantity(0.9572, unit.angstrom)\n",
+    "    theta = Quantity(104.52, unit.degree).to(unit.radian)\n",
+    "\n",
     "    water_reference.add_conformer(\n",
-    "        [[0.0, 0.0, 0.0], [-0.7815, 0.5526, 0.0], [0.7815, 0.5526, 0.0]] * unit.angstrom\n",
+    "        bond_length\n",
+    "        * Quantity(\n",
+    "            [\n",
+    "                [0.0, 0.0, 0.0],\n",
+    "                [-sin(theta / 2), cos(theta / 2), 0.0],\n",
+    "                [sin(theta / 2), cos(theta / 2), 0.0],\n",
+    "            ]\n",
+    "        )\n",
```

Comment:
I'm convinced this is the cause of the years-long energy difference we've been observing

```python
In [1]: from openff.units import *

In [2]: from math import *

In [3]: bond_length = Quantity(0.9572, unit.angstrom)

In [4]: theta = Quantity(104.52, unit.degree).to(unit.radian)

In [5]: bond_length * Quantity(
   ...:     [
   ...:         [0.0, 0.0, 0.0],
   ...:         [-sin(theta / 2), cos(theta / 2), 0.0],
   ...:         [sin(theta / 2), cos(theta / 2), 0.0],
   ...:     ],
   ...: )
Out[5]:
array([[ 0.        ,  0.        ,  0.        ],
       [-0.75695033,  0.58588228,  0.        ],
       [ 0.75695033,  0.58588228,  0.        ]]) <Unit('angstrom')>